### ISSUE TITLE
[DOCS] Create HIRING.md and link to README.md

### ISSUE THREAD

**Issue by tesla809** [COVID-19-electronic-health-system/Corona-tracker#694]
# ⚠️ IMPORTANT: Please fill out this template to give us as much information as possible to consider/implement this update.

### Summary

For our $100 donation or $100/month sponsors, they can post on Hiring.md and have access to our developers who are looking. It will be part of our ethical advertising model. Depending on feedback, we might want to move HIRING.md to the front page of README.md.

The placement will last 1 month for a 1-time donation or stay on-page for as long as the entity is a recurring donor.

1) Please create a file called HIRING.md and link it to a prominent place in the README.md where people can see it.

At the top place of HIRING.md:
tl;dr: Boost your recruiting success while empowering the work on an ambitious and promising Open Source project - for one month, for a tax-deductible donation of $USD 100 via [OpenCollective](https://opencollective.com/coronatracker).

2) Link to README.md in a prominent place at the beginning with eye-catching emojis 
Place a variation of this text on README.md
tl;dr: Boost your recruiting success while empowering the work on an ambitious and promising Open Source project - for one month, for tax-deductible donation of $USD 100 via [OpenCollective](https://opencollective.com/coronatracker).

3) Create a google sheet in our CoronaTracker gDrive to track month and entity donating. 

An example can be:
https://hoodie.martynus.net/sponsoring.html

See this for more information (or ideas!)
https://docs.opencollective.com/help/collectives/funding-options#ethical-advertising

### Motivation
**Why are we doing this?** 
This will help with our fundraising efforts and fund our infrastructure costs.

**What use cases does it support?** 
It will help us fund our costs as we start to scale. It will also help developers on the project land great jobs during these trying times.

**What is the expected outcome?**
We will have a section that hosts posting for jobs by contributors.

### Possible Alternatives

We can place the job placements in the README.md.

### Additional Context

See:
https://docs.opencollective.com/help/collectives/funding-options#ethical-advertising
